### PR TITLE
Animation for Spin button, added middle and bottom row validations

### PR DESCRIPTION
**Pull request by irshadshalu**

### CODE REVIEW

File: tambola/templates/public/manage.html [hunk 1]
```diff
@@ -3,17 +3,29 @@
...
-        <button type="button" class="btn btn-primary" onclick="newGame()">New Game</button>
-        {{game.numbers}}
+			<button type="button" class="btn btn-success btn-circle btn-xl" onclick="nextNumber()">Spin</button>
+		</div>
+		<br>
+		<div class="js-odoo"></div>
+		<div style="display: none" id="cur_num">{{game.numbers.split(",")[0]}}</div>
+		<br>
+		<div class="prev-nums">
+        	Previous: {{",".join(game.numbers.split(",")[1:])}}
```
Comment (sidharthv96) [inline tambola/templates/public/manage.html]: formatting

File: assets/css/style.css [hunk 2]
```diff
@@ -84,3 +84,37 @@ footer .footer-nav {
...
+  fill: #111;
+  margin-left: 6%;
+  display: block;
+  justify-content: center;
+  align-items: center;
+ }
+
+ .prev-nums {
+    font-size: large;
+    text-shadow: 1px 1px 1px rgba(0, 0, 0, 0.33);
```
Comment (sidharthv96) [inline assets/css/style.css]: can you remove this shadow?

File: tambola/templates/public/manage.html [hunk 3]
```diff
@@ -3,17 +3,29 @@
...
 
-        <h1>Game {{game.id}}</h1>
-        <button type="button" class="btn btn-primary" onclick="nextNumber()">Generate</button>
-        <button type="button" class="btn btn-primary" onclick="newGame()">New Game</button>
-        {{game.numbers}}
+			<button type="button" class="btn btn-success btn-circle btn-xl" onclick="nextNumber()">Spin</button>
+		</div>
+		<br>
+		<div class="js-odoo"></div>
+		<div style="display: none" id="cur_num">{{game.numbers.split(",")[0]}}</div>
```
Comment (sidharthv96) [inline tambola/templates/public/manage.html:14]: does this cause issue when there arent any numbers?

File: tambola/templates/public/manage.html [hunk 4]
```diff
@@ -3,17 +3,29 @@
...
+	<br>
+    <div class="col-md-8" align="center">
+		<div>
+	        <h1>Game {{game.id}}</h1>
 
-        <h1>Game {{game.id}}</h1>
-        <button type="button" class="btn btn-primary" onclick="nextNumber()">Generate</button>
-        <button type="button" class="btn btn-primary" onclick="newGame()">New Game</button>
-        {{game.numbers}}
+			<button type="button" class="btn btn-success btn-circle btn-xl" onclick="nextNumber()">Spin</button>
```
Comment (sidharthv96) [inline tambola/templates/public/manage.html]: Can you move the button below the spinner?

**Review by sidharthv96 — CHANGES_REQUESTED** [review]

File: tambola/templates/public/manage.html (hunk 3 above)
Comment (irshadshalu) [reply tambola/templates/public/manage.html:14]: Nope, seems like it just ignores it.

**Review by sidharthv96 — APPROVED** [review]

**Pull request merged**